src/Config/CS_SDK.props

```diff
@@ -1,8 +1,8 @@
 <Project>
   <PropertyGroup>
-    <Platforms>NET60_Windows;NET60_Linux;</Platforms>
+    <Platforms>AnyCPU;NET60_Linux;</Platforms>
```

Comment:
Do we remove NET60_Linux from here as well or is this props file shared with the other Core solution that supports Linux?

Reply:
Yes, it is shared.